www/src/pages/pt/deployment/docker.md

```diff
@@ -0,0 +1,213 @@
+
+# ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
+
+COPY --from=builder /app/next.config.mjs ./
+COPY --from=builder /app/public ./public
+COPY --from=builder /app/package.json ./package.json
+
+COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
+COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
+
+USER nextjs
+EXPOSE 3000
+ENV PORT 3000
+
+CMD ["node", "server.js"]
+
+```
+
+> **_Notas_**
+>
+> - _A emulação de `--platform=linux/amd64` pode não ser necessária após a mudança para o Node 18._
+> - _Consulte [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) para entender por que `libc6-compat` pode ser necessário._
+> - _Next.js coleta [dados anônimos de telemetria sobre uso geral](https://nextjs.org/telemetry). Remova o comentário da primeira instância de `ENV NEXT_TELEMETRY_DISABLED 1` para desabilitar a telemetria durante a compilação. Remova o comentário da segunda instância para desabilitar a telemetria durante o tempo de execução._
+
+</div>
+</details>
+
+## Crie e execute a imagem localmente
+
+Crie e execute esta imagem localmente com os seguintes comandos:
+
+```bash
+docker build -t ct3a-docker --build-arg NEXT_PUBLIC_CLIENTVAR=clientvar .
+docker run -p 3000:3000 -e DATABASE_URL="database_url_goes_here" ct3a-docker
+```
+
+Abra [localhost:3000](http://localhost:3000/) para ver sua aplicação rodando.
+
+## Docker Compose
+
+Você também pode usar o Docker Compose para criar a imagem e executar o contêiner.
+
+<details>
+    <summary>
+      Siga as etapas 1 a 4 acima, clique aqui e inclua o conteúdo em <code>docker-compose.yml</code>:
+    </summary>
+<div class="content">
+
+```yaml
+version: "3.9"
+services:
+  app:
+    platform: "linux/amd64"
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        NEXT_PUBLIC_CLIENTVAR: "clientvar"
+    working_dir: /app
+    ports:
+      - "3000:3000"
+    image: t3-app
+    environment:
+      - DATABASE_URL=database_url_goes_here
+```
+
+Execute isso usando o comando `docker compose up`:
+
+```bash
+docker compose up
+```
+
+Abra [localhost:3000](http://localhost:3000/) para ver sua aplicação rodando.
+
+</div>
+</details>
+
+## Deploy na Railway
+
+Você pode usar uma PaaS como [Railway's](https://railway.app) e seus [deploys de Dockerfile](https://docs.railway.app/deploy/dockerfiles) automáticos para fazer deploy das suas aplicações. Se você tiver o [CLI da Railway instalada](https://docs.railway.app/develop/cli#install), poderá implantar sua aplicação com os seguintes comandos:
+
+```bash
+railway login
+railway init
+railway link
+railway up
+railway open
+```
+
+Vá para "Variáveis" e inclua seu `DATABASE_URL`. Em seguida, vá para "Configurações" e selecione "Gerar domínio". Para ver um exemplo em execução no Railway, visite [ct3a-docker.up.railway.app](https://ct3a-docker.up.railway.app/).
```

Comment:
```suggestion
Vá para "Variables" e inclua seu `DATABASE_URL`. Em seguida, vá para "Settings" e selecione "Generate Domain". Para ver um exemplo em execução no Railway, visite [ct3a-docker.up.railway.app](https://ct3a-docker.up.railway.app/).
```

I changed back the words to English because the user interface of Railway is in English.